api/controllers/UserController.js

```diff
@@ -252,18 +252,20 @@ module.exports = {
 
   findForNetwork: function (req, res) {
     var total
-
-    Community.query().where('network_id', req.param('networkId')).select('id')
-    .then(rows => _.pluck(rows, 'id'))
-    .then(ids => Search.forUsers({
-      communities: ids,
-      limit: req.param('limit') || 20,
-      offset: req.param('offset') || 0
-    }).fetchAll({withRelated: ['skills', 'organizations']}))
-    .tap(users => total = (users.length > 0 ? users.first().get('total') : 0))
-    .then(users => users.map(UserPresenter.presentForList))
-    .then(list => ({people_total: total, people: list}))
-    .then(res.ok, res.serverError)
+    Network.find(req.param('networkId'))
+    .then(network =>
```

Comment:
it's not necessary to nest this whole thing in the first `.then` -- it can stay pretty much the same as it was, except for wrapping the first `Community.query` line in a `.then` callback after `Network.find`